tools/docker-compose/bootstrap_development.sh

```diff
@@ -25,6 +25,7 @@ fi
 
 # Make sure that the UI static file directory exists, Django complains otherwise.
 mkdir -p /awx_devel/awx/ui/build/static
+mkdir -p /awx_devel/awx/ui_next/build/awx
```

Comment:
Since we added a default template, I think this line should no longer be needed.

Reply:
i ended up moving the placeholder index_awx.html out because 
![image](https://user-images.githubusercontent.com/44379968/226410678-333dfada-a239-419a-8e52-8c6e23cbea29.png)

but great idea and thanks for the commit